src/objective-c/GRPCClient/GRPCCall+Tests.h

```diff
@@ -63,4 +63,19 @@
  * cache.
  */
 + (void)resetHostSettings;
+
+/**
+ * Enables logging of op batches. Memory consumption increases as more ops are logged.
+ */
++ (void)enableOpBatchLog:(BOOL)enabled;
+
+/**
+ * Add an op batch to log.
+ */
++ (void)addOpBatchToLog:(NSArray *)batch;
+
+/**
+ * Obtain the logged op batches. Invoking this method will clean the log.
+ */
++ (NSArray *)obtainAndCleanOpBatchLog;
```

Comment:
You're right, the best we can do is put it under `GRPCClient/testing/` or `GRPCClient/internal_testing/`, and `NSLog` on `enableOpBatchLog` and `obtainAndCleanOpBatchLog` that these should be used only for testing gRPC itself. And let's make it very clear in the docs of each of these methods that they're not part of the public interface of the library.